tests/python/pants_test/engine/test_mapper.py

```diff
@@ -145,21 +145,33 @@ def unhydrated_structs(build_file_addresses):
   yield UnhydratedStructs(uhs)
 
 
-class AddressMapperTest(unittest.TestCase, SchedulerTestBase):
+class AddressMapperTest(TestBase, unittest.TestCase):
```

Comment:
This class might be a bit of a special case: it doesn't use the `BuildFileAliases`, or anything else from the `legacy` package. It's intentionally testing other parsers in an isolated environment. So it might be one of the few cases that should actually use a scheduler directly.